datafaker-gen-examples/datafaker-gen-bigquery/pom.xml

```diff
@@ -161,4 +161,48 @@
         </plugins>
     </build>
 
+
+    <profiles>
+        <profile>
+            <id>default</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-surefire-plugin</artifactId>
+                        <version>3.3.0</version>
+                        <configuration>
+                            <excludes>
+                                <exclude>**/net/datafaker/datafaker_gen/sink/**/*Test.java</exclude>
```

Comment:
Do we really need to exclude all or just `BigQueryConnectionConfigurationTest` is enough?

Reply:
No. I exclude just one BigQuerySinkIntegrationTest.